packages/frontend/src/translations/en.global.json

```diff
@@ -1607,33 +1608,33 @@
     "stakingPoolUnstaked": "This NEAR is currently in a staking pool but unstaked. It could for example be in a pending release state.",
     "success": "Success",
     "swap": {
+        "available": "Available: ${amount} ${symbol}",
```

Comment:
no way, do you sort it by alphabet order?). I don't care about it, so it's easy to copy-paste-translate for all locales.